server/eclipse-project/src/main/java/us/freeandfair/corla/asm/ASMTransitionFunction.java

```diff
@@ -207,10 +207,11 @@ public ASMTransition value() {
     O(new ASMTransition(ROUND_IN_PROGRESS,
                         ROUND_COMPLETE_EVENT,
                         WAITING_FOR_ROUND_SIGN_OFF)),
-    /* We probably want this transition eventually, but not for CDOS
+    /* We probably want this transition eventually, but not for CDOS */
+    // idk, this is what the smoke test is looking for, so let's try it
     EARLY(new ASMTransition(ROUND_IN_PROGRESS,
                             RISK_LIMIT_ACHIEVED_EVENT, 
-                            AUDIT_COMPLETE)), */
+                            AUDIT_COMPLETE)),
```

Comment:
Are we planning to land this in master? This seems like a significant change that would warrant some more testing... I can't point to anything concrete but things like clients continuing to submit ACVRs, possibly with discrepancies...

Reply:
nope I missed this - thanks!